miniupnpd/miniupnpd.conf

```diff
@@ -33,163 +38,168 @@
 # Specify STUN UDP port, by default it is standard port 3478.
 #ext_stun_port=3478
 
-# LAN network interfaces IPs / networks
-# There can be multiple listening IPs for SSDP traffic, in that case
-# use multiple 'listening_ip=...' lines, one for each network interface.
-# It can be IP address or network interface name (ie. "eth0")
+# Internal LAN network interfaces
+################################################################################
+# LAN network interface name or IPv4/netmask
 # It is mandatory to use the network interface name in order to enable IPv6
+# Can be specified more than once for multiple network interfaces
 # HTTP is available on all interfaces.
+# When MULTIPLE_EXTERNAL_IP is disabled, you can list associated network
+# interfaces (for bridges):
+#   listening_ip=bridge0 em0 wlan0
 # When MULTIPLE_EXTERNAL_IP is enabled, the external IP
 # address associated with the subnet follows. For example:
-#  listening_ip=192.168.0.1/24 88.22.44.13
-# When MULTIPLE_EXTERNAL_IP is disabled, you can list associated network
-# interfaces (for bridges)
-#  listening_ip=bridge0 em0 wlan0
-#listening_ip=192.168.0.1/24
-#listening_ip=10.5.0.0/16
+#   listening_ip=192.168.0.1/24 88.22.44.13
 #listening_ip=eth0
-# CAUTION: mixing up WAN and LAN interfaces may introduce security risks!
-# Be sure to assign the correct interfaces to LAN and WAN and consider
-# implementing UPnP permission rules at the bottom of this configuration file
-
-# Port for HTTP (descriptions and SOAP) traffic. Set to 0 for autoselect.
-#http_port=0
-
-# Port for HTTPS. Set to 0 for autoselect (default)
-#https_port=0
+#listening_ip=eth1
+#listening_ip=192.168.0.1/24
+#listening_ip=192.168.1.1/24
 
-# Path to the UNIX socket used to communicate with MiniSSDPd
-# If running, MiniSSDPd will manage M-SEARCH answering.
-# default is /var/run/minissdpd.sock
-#minissdpdsocket=/var/run/minissdpd.sock
+# General daemon configuration and enabled protocols
+################################################################################
+# Lease file location
+#lease_file=/var/run/miniupnpd.leases
```

Comment:
The `/var/log/upnp.leases` was changed to the `/var/run/miniupnpd.leases`.
This makes it same as a default in OpenWrt which it good but maybe it's OpenWrt should change its the default instead.
The systemd has an option to create an own service folder under `/var/run/` https://serverfault.com/a/779648

So maybe it would be better to change it to `/var/run/miniupnpd/upnp.leases`.

Anyway the change of defaults must be made in a separate commit and documented in the commit message to make it easier to understand what was changed and create a changelog.

Reply:
yes, since it's commented then it should be less risky to change it. Still it should be changed in a separate commit as for me. I feel uncomfortable while trying to understand what was changed and if it's important or not.